lib/pages/home_page.dart

```diff
@@ -51,7 +56,7 @@ class _HomePageState extends State<HomePage> {
       ),
       floatingActionButton: FloatingActionButton(
         heroTag: 'fab',
-        child: Icon(EvaIcons.plusOutline),
+        child: Icon(EvaIcons.plusCircleOutline),
```

Comment:
Why the icon is changed?

Reply:
It shows "The getter 'plusOutline' isn't defined for the type 'EvaIcons'."
![image](https://user-images.githubusercontent.com/65543606/135666663-a53e8b26-31b7-4d6c-95b1-2f39d9f37b50.png)

Reply:
That's strange

Reply:
Done.

Reply:
I meant 'PlusOutline' to 'plus'. not 'plusCircle'.

Reply:
Sure, 
I have made the changes 'checkmarkOutline' and 'plusOutline' added as the icon.